packages/card/src/Card.tsx

```diff
@@ -18,16 +17,20 @@ const containerStyle = css`
   }
 `;
 
-function Card<T extends React.ReactNode>(props: BoxProps<T>) {
-  const rest = omit(props as any, ['className']);
-
-  return <Box className={cx(containerStyle, props.className)} {...rest} />;
+interface CardProps {
+  className?: string;
```

Comment:
Maybe I'm missing something, but isn't className also defined as an optional string from the implicit props in ExtendableBox?

Reply:
Chatted offline!